transport-security.md

```diff
@@ -44,7 +44,7 @@ Even while remaining client agnostic, clients can be classified in four major gr
 This section contains elements that apply to the generic classes of clients listed above. Although not every client implementation has a need for all the specifications referenced below, a client agnostic API SHOULD provide these to facilitate any client to implement relevant security controls.
 
 Most specifications referenced in this section are applicable to the first three classes of clients listed above.
-Security considerations for native applications are provided in [OAUth2 for Native Apps]](https://tools.ietf.org/html/rfc8252), much of which can help non-OAuth2 based implementations as well.
+Security considerations for native applications are provided in [[[rfc8252]]], much of which can help non-OAuth2 based implementations as well.
```

Comment:
![image](https://github.com/user-attachments/assets/4ffcea4b-de73-4a91-8736-b375c2e0dea6)
Ik zit hier op render en het lijkt alsof "_OAUth2 for Native Apps_" nog steeds in de tekst staat, maar is dat daadwerkelijk het geval met deze changes?

Reply:
Waarschijnlijk moet je de browser hard refreshen. In de laatste versie op https://logius-standaarden.github.io/Publicatie-Preview/API-mod-transport-security/pr-timvdlippe-rfc-feedback/#http-level-security is de tekst voor mij weg.